azure-pipelines/end-to-end-tests.ps1

```diff
@@ -36,6 +36,7 @@ Param(
 )
 
 $ErrorActionPreference = "Stop"
+$ProgressPreference = "SilentlyContinue"
```

Comment:
This fixes output in the terminal being very broken from overlapping progress bars from every Copy-Item and Remove-Item in the overall harness introduced in PowerShell 7.5.x